# disease_outbreak_detection

adi_gsr_test.py needs have the savePath variable modified so that the output points
to the proper file directory, after that it should work